### PR TITLE
Repo rename change

### PR DESCRIPTION
**Pull request by josmo**
@bradrydzewski I think this fits more in line with what you envisioned from https://github.com/drone/drone-cli/pull/61 Let me know if I took a wrong turn ;)

### CODE REVIEW

File: server/repo.go [hunk 1]
```diff
@@ -238,52 +238,86 @@ func RepairRepo(c *gin.Context) {
...
+		destinationRepo.ResetVisibility()
 	}
+	destinationRepo.Counter = currentRepo.Counter
+	destinationRepo.AllowDeploy = currentRepo.AllowDeploy
+	destinationRepo.AllowPull = currentRepo.AllowPull
+	destinationRepo.AllowPush = currentRepo.AllowPush
+	destinationRepo.AllowTag = currentRepo.AllowTag
+	destinationRepo.IsTrusted = currentRepo.IsTrusted
+	destinationRepo.IsGated = currentRepo.IsGated
+	destinationRepo.IsActive = currentRepo.IsActive
```
Comment (josmo) [inline server/repo.go]: @bradrydzewski I noticed that isActive doesn't get set from remote.Activate  I imagine I should disable the currentRepo and save that change?

**Comment by bradrydzewski** [comment]
thanks, let me think about this a bit more and get back to you :)

**Comment by josmo** [comment]
sounds good. Let me know if a different route seems more compelling :)

**Comment by bradrydzewski** [comment]
one additional item we should set:

```
destinationRepo.Hash = currentRepo.Hash
```

**Comment by josmo** [comment]
@bradrydzewski I added Hash. I did wonder if I should disable the currentRepo
```
currentRepo.IsActive = false
```
otherwise it stays in the database as active..  Not sure if that's a big deal or not.

**Comment by josmo** [comment]
@bradrydzewski Just checking what you were thinking about this. I'm about to rename a bunch of repos namespace and would hold off making a custom build of drone if it gets merged in :)

**Comment by bradrydzewski** [comment]
closing since renaming was built-in to the sync process in 1.0

**Pull request closed without merging**